spec/requests/trains_request_spec.rb

```diff
@@ -29,4 +49,92 @@
       expect(JSON.parse(response.body)).to eq(expected_response)
     end
   end
+
+  describe 'POST /v1/trains' do
+    context 'with valid attributes' do
+      it 'returns http status 201' do
+        post v1_trains_path params: { train: expected_response }
+        expect(response).to be_created
+        expect(JSON.parse(response.body)).to eq({"message"=>"#{train.name} successfully created"})
```

Comment:
we should verify a train was actually created.
we are only verifying that the messages and status from the server are correct, but not that we have actually created a new record

Reply:
can i use `count` for this?

Reply:
probably